src/components/screens/Home/NearByUserProfileScreen.js

```diff
@@ -0,0 +1,89 @@
+import {Image, StyleSheet, View, Text} from 'react-native';
+import {useTheme} from '@react-navigation/native';
+import React from 'react';
+import ProfilePicturesCarousel from './NearByUserProfilePicturesCarousel';
+import ApproachText from '../../ApproachText';
+import {ScrollView} from 'react-native-gesture-handler';
+import InterestsLabelsList from './InterestsLabelsList';
+import Location from '../../../../assets/icons/location.png';
+import ApproachButton from '../../ApproachButton';
+import Message from '../../../../assets/icons/dms.png';
+
+export function NearByUserScreen(props) {
+  const colors = useTheme();
+  const user = props.user;
+  const profilePictures = user.photos;
+  const header = `${user.name}, ${user.age}`;
+  return (
+    <ScrollView style={{...styles.container, backgroundColor: colors.white}}>
+      <ProfilePicturesCarousel images={profilePictures} />
+      <ApproachText style={{...styles.name}} children={header} />
```

Comment:
I think it's better to stick with passing the content of `ApproachText` like below:
```suggestion
      <ApproachText style={{...styles.name}}>
          {user.name}, {user.age}
      </ApproachText>
```
That's how we use it everywhere else.
Same applies to all other places where you use `children` like this.

Reply:
address